src/adagrams.js

```diff
@@ -1,7 +1,87 @@
+const randomItemsIndex = function(items) {
+  return Math.floor(Math.random()*items.length);
+};
+
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    const letterDist = { 
+      A: 9, B: 2, C: 2, D: 4, E: 12, 
+      F: 2, G: 3, H: 2, I: 9, J: 1, 
+      K: 1, L: 4, M: 2, N: 6, O: 8, 
+      P: 2, Q: 1, R: 6, S: 4, T: 6, 
+      U: 4, V: 2, W: 2, X: 1, Y: 2, 
+      Z: 1 
+    };
+    const letterDistArr = Object.entries(letterDist) // returns [["A", 9], ["B", 2] ...]
+    const allLetters = letterDistArr.map(letter => letter[0].repeat(letter[1])); 
+    let letterPool = allLetters.join('').split('');
```

Comment:
You can't win a pulitzer for it, but this is poetry.

Reply:
I bet with a memo you could get rid of the join on line 18, which would be Very Slick.